src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/IssueFinalDecisionAboutToSubmitHandler.java

```diff
@@ -79,6 +97,11 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             if (sscsCaseData.getSscsFinalDecisionCaseData().getWriteFinalDecisionPreviewDocument() != null) {
                 createFinalDecisionNoticeFromPreviewDraft(preSubmitCallbackResponse);
                 clearTransientFields(preSubmitCallbackResponse);
+                if (eligibleForHearingsCancel.test(sscsCaseData)) {
+                    log.info("Issue Final Decision: HearingRoute ListAssist Case ({}). Sending cancellation message",
+                            sscsCaseData.getCcdCaseId());
+                    hearingMessageHelper.sendListAssistCancelHearingMessage(sscsCaseData.getCcdCaseId());
+                }
```

Comment:
Too much nesting. Seek alternative way to if anything reduce the nested if's here

Reply:
I have refactored to remove the nesting.